source/beamline/BioXAS/BioXASFrontEndShutters.cpp

```diff
@@ -46,7 +45,7 @@ void BioXASFrontEndShutters::setUpstreamPhotonShutter(AMReadOnlyPVControl *newCo
 		upstreamPhotonShutter_ = newControl;
 
 		if (upstreamPhotonShutter_)
-			addShutter(upstreamPhotonShutter_, 1, 4);
+			addShutter(upstreamPhotonShutter_, CLS_SHUTTER_OPEN, CLS_SHUTTER_CLOSED);
```

Comment:
@helfrij and also in this case, we were using the magic number (1, 4). I simply replaced it with the state definitions,